tests/e2e/specFiles/Transactions/Cancel.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { expect } from 'detox';
+
+import { isBeta, createRandomNote, WALLETS_WITH_COINS, DEFAULT_TRANSACTION_PASSWORD } from '../../helpers';
+import app from '../../pageObjects';
+import steps from '../../steps';
+
+const DATA_FOR_TRANSACTIONS = {
+  AMOUNT_TO_SEND: '0.0001',
+};
+
+describe('Transactions', () => {
+  beforeEach(async () => {
+    isBeta() && (await app.onboarding.betaVersionScreen.close());
+    await app.developerRoom.tapOnSkipOnboardingButton();
+    await app.navigationBar.changeTab('wallets');
+  });
+
+  describe('Cancel', () => {
+    describe('@ios @regression', () => {
+      //TODO: Test is disabled due to long transaction confirmations.
```

Comment:
leave an empty space between `//` and the actual comment

Reply:
Done